aea/decision_maker/default.py

```diff
@@ -246,7 +246,7 @@ def apply_transactions(
         """
         new_state = copy.copy(self)
         for tx_message in transactions:
-            new_state._update(tx_message)
+            new_state._update(tx_message)  # pylint: disable=protected-access
```

Comment:
does `_update` has to be private? O/w make it public

Reply:
It can be public. Will fix.